openhtf/io/frontend/__main__.py

```diff
@@ -96,47 +96,41 @@ class StationStore(threading.Thread):
   """
   def __init__(self):
     super(StationStore, self).__init__()
-    self._death_event = threading.Event()
+    self._stop_event = threading.Event()
```

Comment:
right, my implication was that both are oddly named, but it's definitely a nit at best, so i wouldn't worry about fixing it :)